gtsfm/loader/one_d_sfm_loader.py

```diff
@@ -0,0 +1,162 @@
+"""Loader of dataset used in 1DSFM and Colmap paper
+
+The dataset is collected from internet and has EXIF embedded in each image,
```

Comment:
Is it a single dataset, or a collection of datasets?

Reply:
It does contain several sequences, with each sequence as an independent dataset.  This class is designed to load a single sequence, I will modify the description accordingly.